flake.nix

```diff
@@ -1,72 +1,63 @@
 {
   description = ''
-    next-gen framework for building desktop widgets using Python (check the rewrite branch for progress)
+    next-gen framework for building desktop widgets using Python
+    (check the rewrite branch for progress)
   '';
 
   inputs = {
     nixpkgs.url = "github:NixOS/nixpkgs/24.05";
-    nixpkgs-unstable.url = "github:NixOS/nixpkgs/nixos-unstable";
     utils.url = "github:numtide/flake-utils";
   };
 
-  outputs = {
-    nixpkgs,
-    nixpkgs-unstable,
-    utils,
-    ...
-  }:
-    utils.lib.eachDefaultSystem
-    (system: let
-      overlay = final: prev: {
-        pythonPackagesExtensions =
-          prev.pythonPackagesExtensions
-          ++ [
-            (
-              python-final: python-prev: {
-                fabric-widgets = prev.callPackage ./default.nix {};
-              }
-            )
+  outputs =
+    { nixpkgs, utils, ... }:
+    utils.lib.eachDefaultSystem (
+      system:
+      let
+        overlay = final: prev: {
+          pythonPackagesExtensions = prev.pythonPackagesExtensions ++ [
+            (python-final: python-prev: {
+              # `fabric` was taken
+              fabric-widgets = prev.callPackage ./default.nix { };
```

Comment:
> [!NOTE]  
> `fabric` was taken; in any attempt to officially package Fabric either as a standalone application or Python package an available name needs to be chosen.

Reply:
it's okay to use either `python-fabric` so we have a uniform with the AUR package or `fabric-widgets`

Reply:
> it's okay to use either `python-fabric`

Changed in f11ff86 :)